### PR TITLE
Subscription authorization tests

### PR DESCRIPTION
**Pull request by sjanota**
**Description**

Changes proposed in this pull request:

- tests for authorization in subscriptions (working after #8620) are added

**Related issue(s)**

### CODE REVIEW

File: tests/console-backend-service/internal/domain/eventing/eventing_test.go [hunk 1]
```diff
@@ -12,6 +12,8 @@ import (
 	"github.com/kyma-project/kyma/tests/console-backend-service/internal/module"
 
 	"github.com/pkg/errors"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
```
Comment (dariadomagala) [inline tests/console-backend-service/internal/domain/eventing/eventing_test.go:15]: what is it for?

File: tests/console-backend-service/internal/domain/eventing/trigger_test.go [hunk 2]
```diff
@@ -208,7 +211,7 @@ func triggerDetailsFields() string {
 		name
     	namespace
 		broker
-    	filterAttributes
+		filterAttributes
```
Comment (dariadomagala) [inline tests/console-backend-service/internal/domain/eventing/trigger_test.go:214]: indentations in this whole query look weird

File: tests/console-backend-service/internal/domain/eventing/eventing_test.go (hunk 1 above)
Comment (sjanota) [reply tests/console-backend-service/internal/domain/eventing/eventing_test.go:15]: It allows to run your the tests locally against cluster running on GCP

File: tests/console-backend-service/internal/domain/eventing/trigger_test.go (hunk 2 above)
Comment (sjanota) [reply tests/console-backend-service/internal/domain/eventing/trigger_test.go:213]: fixed

**Pull request merged**